modules/ctrl_dbus/gen.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+gdbus-codegen --generate-c-code baresipbus --c-namespace DBus \
+    --interface-prefix com.creytiv. com.creytiv.Baresip.xml
```

Comment:
please note that creytiv.com domain has been closed.
you should probably use a different domain

Reply:
com.baresip.Baresip.xml ?

Reply:
Have no better idea. If you have, I'll change it later.

Reply:
maybe: `com.github.Baresip.xml` ?

Reply:
ok. definitely better.